Dockerfile

```diff
@@ -7,8 +7,8 @@ EXPOSE 80
 ENV MATTERMOST_VERSION 3.1.0
 
 ADD https://releases.mattermost.com/${MATTERMOST_VERSION}/mattermost-team-${MATTERMOST_VERSION}-linux-amd64.tar.gz /
-RUN tar -C / -xzf /mattermost-team-${MATTERMOST_VERSION}-linux-amd64.tar.gz --no-same-owner \
-	&& rm -rf /mattermost-team-${MATTERMOST_VERSION}-linux-amd64.tar.gz
+RUN tar -C / -xzf /mattermost-team-${MATTERMOST_VERSION}-linux-amd64.tar.gz --no-same-owner && \
+    rm  -f /mattermost-team-${MATTERMOST_VERSION}-linux-amd64.tar.gz
```

Comment:
-f          Attempt to remove the files without prompting for confirmation, regardless of the file's permis-
                 sions.  If the file does not exist, do not display a diagnostic message or modify the exit status
                 to reflect an error.  The -f option overrides any previous -i options.